docs/ref/contrib/admin/index.txt

```diff
@@ -2507,11 +2507,18 @@ Templates can override or extend base admin templates as described in
     * ``site_title``: :attr:`AdminSite.site_title`
     * ``site_url``: :attr:`AdminSite.site_url`
     * ``has_permission``: :meth:`AdminSite.has_permission`
+    * ``available_apps``: a list of applications from the
```

Comment:
"a list of applications" is too vague, I think. Looking at the test, it seems to return much more than a list of simple items. Maybe an example of the structure that is returned would be appropriate?